apps/ai-content-generator/src/configs/launch-darkly/ldConfig.ts

```diff
@@ -0,0 +1,16 @@
+const ldConfig = {
+  clientSideID: `${import.meta.env.VITE_LAUNCHDARKLY_CLIENT_ID}`,
+  deferInitialization: false,
+  flags: {
+    'integrations-ai-content-generator-v-2': true,
+  },
+  options: {
+    bootstrap: localStorage,
+  },
+};
+
+type ldConfigType = { integrationsAiContentGeneratorV2: boolean };
```

Comment:
Okay this is like a magic thing that the LD client does at compile time where it automatcally converts the feature flag `integrations-ai-content-generator-v-2` to that camelCase type? Interesting.

Reply:
Yeah they magically convert it for us. We can disable that in the config with a flag, and it'll spit out `integerations-ai-...`